doc/linux/build_instructions.md

```diff
@@ -73,3 +73,8 @@ mkdir build && cd build
 cmake -DWITH_EXAMPLES=on -DCMAKE_PREFIX_PATH="/opt/glog;/opt/protobuf;/opt/websockets" ..
 make
 ```
+
+## SDK with bindings
+
+- Please check the readme files for each type of binding in the <a href="https://github.com/analogdevicesinc/aditof_sdk/tree/master/bindings"> bindings directory </a>.
```

Comment:
Is the hyperlink really necessary?

Reply:
I thought it is easier for the user this way. I can take it out if you think it is more confusing than helpful.

Reply:
But what is easier?

Reply:
to get to the right directory

Reply:
I meant the using of 'href'. You can easily do this without html.

Reply:
ok, solved.